package.json

```diff
@@ -612,27 +624,22 @@
                 {
                     "command": "aws.s3.createFolder",
                     "when": "view == aws.explorer && viewItem =~ /^(awsS3BucketNode|awsS3FolderNode)$/",
-                    "group": "0@2"
+                    "group": "1@1"
                 },
                 {
                     "command": "aws.copyName",
                     "when": "view == aws.explorer && viewItem =~ /^(awsRegionFunctionNode|awsCloudFormationFunctionNode|awsStateMachineNode|awsCloudFormationNode|awsS3BucketNode|awsS3FolderNode|awsS3FileNode)$/",
-                    "group": "1@1"
+                    "group": "2@1"
                 },
                 {
                     "command": "aws.s3.copyPath",
                     "when": "view == aws.explorer && viewItem =~ /^(awsS3FolderNode|awsS3FileNode)$/",
-                    "group": "1@2"
+                    "group": "2@2"
                 },
                 {
                     "command": "aws.copyArn",
                     "when": "view == aws.explorer && viewItem =~ /^(awsRegionFunctionNode|awsCloudFormationFunctionNode|awsStateMachineNode|awsCloudFormationNode|awsS3BucketNode|awsS3FolderNode|awsS3FileNode)$/",
-                    "group": "1@3"
-                },
-                {
-                    "command": "aws.s3.createBucket",
-                    "when": "view == aws.explorer && viewItem == awsS3Node",
-                    "group": "1@4"
+                    "group": "2@3"
```

Comment:
Can you clarify the suggestion? This is what it looks like. I think it's reasonable since all the "create-like" stuff goes at the top and all the copy stuff is just below it. Perhaps we can revisit at a later time

![image](https://user-images.githubusercontent.com/1410453/83079353-8052b600-a030-11ea-8fed-16fad12d68f2.png)

Reply:
It's a nit, if everything else has:
Copy Name
Copy ARN
in that order, I'd recommend keeping it that way and moving Copy Path (either underneath instead of between the two or to a different section, probably underneath if anything)

Reply:
Oh, sure. I think I put that because I saw ARN as the least valuable ID to copy (since S3 makes little use of them).

But there's merit in keeping the ordering consistent with other nodes.